docs/pages/dsDisplay/ds-display_L3_Test-Procedure.md

```diff
@@ -0,0 +1,305 @@
+# Device Settings Display L3 Test Case and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Setting Up Test Environment](#setting-up-test-environment)
+- [Test Case Procedure](#test-case-procedure)
+  - [dsDisplay_test01_VerifyDisplayConnectCallBackTest.py](#dsdisplay_test01_verifydisplayconnectCallBackTestpy)
+  - [dsDisplay_test02_TestVerifyDisplayEdid.py](#dsdisplay_test02_testverifydisplayEdidpy)
+  - [dsDisplay_test03_AspectRatioVerificationTest.py](#dsdisplay_test03_aspectratioverificationtestpy)
+  - [dsDisplay_test04_TestVerifyDisplayEdidBytes.py](#dsdisplay_test04_testverifydisplayEdidbytespy)
+
+## Overview
+
+This document describes the L3 Test case Procedure Documentation for the Device Settings Display module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL` \- Hardware Abstraction Layer, may include some common components
+- `UT`  \- Unit Test(s)
+- `OEM` \- Original Equipment Manufacture
+- `SoC` \- System on a Chip
+- `HDMI`\- High-Definition Multimedia Interface
+- `EDID`\- Extended Display Identification Data
+- `RAFT`\- Rapid Automation Framework for Testing
+- `DVI` \- Digital Video Interface
+- `Y`   \- yes supported
+- `NA`  \- Not Supported
+
+### References
+
+- `High Level Test Specification` - [dsDisplay High Level TestSpec](ds-display-high-Level_TestSpec.md)
+
+- `dsDisplay L3 Low Level Test Specification` - [dsDisplay L3 Low Level TestSpec](ds-display_L3_Low-Level_TestSpecification.md)
+
+## Setting Up Test Environment
+
+To execute `HAL` `L3` Python test cases, need a Python environment. Follow these steps mentioned in [HPK Public Documentation](https://github.com/rdkcentral/rdk-hpk-documentation/blob/main/README.md)
+
+### Update Configuration Files
+
+#### Rack Configuration File
+
+Example Rack configuration File: `ut/host/tests/configs/example_rack_config.yml`
```

Comment:
Establish a proper link to the document.  Need to set the path correctly

Reply:
Done